Dockerfile

```
@@ -69,10 +69,10 @@ RUN apt-add-repository ppa:brightbox/ruby-ng \
 ENV PYENV_ROOT=/usr/local/.pyenv \
   PATH="/usr/local/.pyenv/bin:$PATH"
 RUN git clone https://github.com/pyenv/pyenv.git /usr/local/.pyenv \
-  && cd /usr/local/.pyenv && git checkout 9ee109b66148bc39a685926050b7b56cb4bb184b && cd - \
-  && pyenv install 3.9.2 \
+  && cd /usr/local/.pyenv && git checkout 32ba5d5e617b7d7fb1c9b79c1c0e8f1cfe642cd1 && cd - \
```

Version 1.2.25 was just released
```suggestion
  && cd /usr/local/.pyenv && git checkout v1.2.25 && cd - \
```

---

Thanks!